tests/plugins/test_plugins_manager_www.py

```diff
@@ -83,3 +85,66 @@ def test_menu_links(self):
         [menu_link] = [ml for ml in category.get_children()
                        if isinstance(ml, MenuLink)]
         self.assertEqual('Test Menu Link', menu_link.name)
+
+
+class MockPluginA(AirflowPlugin):
+    name = 'plugin-a'
+
+
+class MockPluginB(AirflowPlugin):
+    name = 'plugin-b'
+
+
+class MockPluginC(AirflowPlugin):
+    name = 'plugin-c'
+
+
+class PluginsTestEntrypointLoad(unittest.TestCase):
```

Comment:
are these two unit test only worked for old UI? If not I would suggest to move to test_plugins_manager_rbac.py once we delete the old UI code, we will remove this unit test file as well.

Reply:
I'm not sure, as I haven't touched on the UI parts of the code base yet.

This code is not really anything to do with the UI as far as I'm aware. Or at least, it's not specific to one UI or the other. When I pulled the above PR changes in, this is where my code was move to, so the old test/plugin_manager.py file was renamed to this.

Reply:
From what I've seen so far, the plugin manager loads the plugins, and dumps their respective components (hooks, operators, blueprints, etc) into lists and then it's up to other components to decide which lists to reference.

I'm happy to put in another PR to move it so keep thing Clean™, but I'm not sure where it'd go. Are there plans for the old UI to be deprecated at a time in the future? That could solve the problem.